doc/contributing/security-model-strategy.md

```diff
@@ -0,0 +1,81 @@
+# Security Model Strategy
+
+A clear security model, with features like permissions and policy enforcement,
+is a
+[top technical priority](https://github.com/nodejs/node/blob/master/doc/contributing/technical-priorities.md#permissionspoliciessecurity-model)
+of Node.js.
+
+This document provides an overview of our high-level strategy for
+making progress as well as information on our current implementations
+as a starting point for contributors.
```

Comment:
This paragraph can be deleted. The document is short and its contents are obvious from the title (and if not--change the title?).

Reply:
Done